Source/Core/DolphinQt2/Config/FilesystemWidget.cpp

```diff
@@ -115,6 +124,21 @@ void FilesystemWidget::PopulateDirectory(int partition_id, QStandardItem* root,
     PopulateDirectory(partition_id, root, file_system->GetRoot());
 }
 
+static u64 GetDirectorySize(const DiscIO::FileInfo& directory)
```

Comment:
My comment from the previous PR applies (I've edited it slighly):

> This function would be appropriate to have as a member function of `FileInfo` rather than having it in DolphinQt2. It could also be nice if the function returns the file size when called for a regular file and has a name that reflects that (maybe `GetTotalSize`?)